conversion/conversion.go

```diff
@@ -79,6 +79,10 @@ const (
 	// Target db for which schema is being generated.
 	TARGET_SPANNER               string = "spanner"
 	TARGET_EXPERIMENTAL_POSTGRES string = "experimental_postgres"
+
+	// Database dialects constants supported by spanner.
+	DIALECT_SPANNER               string = "GOOGLE_STANDARD_SQL"
+	DIALECT_EXPERIMENTAL_POSTGRES string = "POSTGRESQL"
```

Comment:
Should we fetch this from adminpb? If yes, what is the field? Because I didnt see them in any of these [files](https://github.com/hengfengli/go-genproto/blob/master/googleapis/spanner/admin/database/v1/). 
[Found](https://github.com/googleapis/googleapis/blob/spanner-type-annotation/google/spanner/admin/database/v1/common.proto#L103) them in the main repo tho